velox/exec/fuzzer/ReferenceQueryRunner.h

```diff
@@ -35,6 +35,23 @@ class ReferenceQueryRunner {
       const std::string& sql,
       const std::vector<RowVectorPtr>& input,
       const RowTypePtr& resultType) = 0;
+
+  /// Returns true if the underlying Query Runner supports
+  /// returning the results of a query in Velox Vector format. If supported
+  /// then 'executeVector' can be called to get results as Velox Vector.
+  virtual bool supportsVeloxVectors() const {
+    return false;
+  }
+
+  /// Executes SQL query returned by the 'toSql' method using 'input' data
```

Comment:
nit: It might be clearer to explain that this method is same as 'execute' except for the format of the results.

```
/// Same as 'execute', but returns results as RowVectorPtr. The caller much ensure that 'supportsVeloxVectorResults' returns 'true'.
```